eventloop/eventloop.go

```diff
@@ -228,6 +240,10 @@ func (loop *EventLoop) RunOnLoop(fn func(*goja.Runtime)) {
 	loop.addAuxJob(func() { fn(loop.vm) })
 }
 
+func (loop *EventLoop) JobRemainCount() int32 {
```

Comment:
This will be only safe to call from inside the loop, because it does non-synchronised access to loop.jobCount. And again, the usefulness of this is questionable because it provides a snapshot of the value at some unspecified point in time (unless called from the loop).

Reply:
I need to judge whether there are any remaining tasks after stop in order to perform pooling related operations

Reply:
> A better way could be making the `Stop` method return it. If you still need it feel free to open a separate PR.

This is a great idea! I may need this feature